main.c

```diff
@@ -0,0 +1 @@
+
```

Comment:
CIAL see style.md

File names should end in `no`

Reply:
I think that this change could get some more documentation. Really, please _always_ add some reasoning to your PRs. It's frustrating having to guess why you made a particular change.

Apart, I question the choice of programming language. The code / comment (=no-code) ratio in C is generally too high. It has 8 types of comments! Maybe use [Enterprise](https://github.com/joaomilho/Enterprise)™ instead? Also, Enterprise encourages good practices, i.e. not running the code, by having the `unnecessary` keyword, which prevents a file from being run.

Reply:
This doesn't pass the linter!
![image](https://user-images.githubusercontent.com/304936/108735948-c7c38f80-7539-11eb-813d-74208f031fb0.png)